### ISSUE TITLE
partial batch normalization in action recognition model

### ISSUE THREAD

**Issue by yezqNLP** [dmlc/gluon-cv#1250]
Hi, I found partial batch normalization `patial-bn` in action recognition model, I know it means to `whether to freeze bn layers except the first layer`. However, I don't know why we are doing this? I tried to google `partial batch normalization`, but  I can't find satisfied answer. 
ps. This issue is not a bug report or problem of your code, so if it's busy for you, you can give me a reference and I will read it myself. : )
Thanks! 
@bryanyzhu

**Comment by bryanyzhu** [comment]
Hi, partial BN is used when your dataset is small and your model is deep. For example, if you train a ResNet101 model on UCF101 directly, you will encounter serious overfitting problem. In order to overcome this issue, Limin Wang etal. proposes this partial BN technique in their ECCV 2016 paper (TSN, https://arxiv.org/pdf/1608.00859.pdf), you can read more detail there. It is in `network training` from page 7 to 8. 

Now we have larger datasets like Kinetics400 or Sthsthv2, we don't need to do partial BN anymore. It's just a regularization technique for train or finetune on small datasets.

**Comment by yezqNLP** [comment]
Thanks so much!
